scripts/do.sh

```diff
@@ -16,11 +16,20 @@ https://github.com/getsentry/devenv#install
     exit 1
 fi
 
-if ! [[ "$VIRTUAL_ENV" -ef "${HERE}/../.venv" ]]; then
-    echo "
-Your virtualenv isn't activated. You need to successfully run 'direnv allow'.
+if [[ ${GETSENTRY:-0} -eq 1 ]]; then
```

Comment:
see https://github.com/getsentry/getsentry/pull/13540

now that i'm revisiting all of this it's pretty annoying how much getsentry make has drifted...

really need to think about how we can move as much of this stuff over to devenv